d3-map-of-world/Lessons/0070_index.js

```diff
@@ -0,0 +1,14 @@
+// Setup
+
+
+/*
+As noted in the first lesson, D3 is designed to work with common web standards,
```

Comment:
Since the note about web standards will be moved to the intro page, perhaps this could read:

```
This project will use D3 to draw a map with SVG, which you'll add to your page with this script. SVG elements have `height` and `width` attributes, so the browser knows the size and how to display them.

Using the `const` keyword, declare an `svgWidth` variable and set it to `1000`, and declare an `svgHeight` variable and set it to `700`.
```